model/metric.go

```diff
@@ -318,21 +315,15 @@ func EscapeName(name string, scheme EscapingScheme) string {
 		}
 		escaped.WriteString("U__")
 		for i, b := range name {
-			if isValidLegacyRune(b, i) {
+			if b == '_' {
+				escaped.WriteString("__")
```

Comment:
Something rings a bell that this was decided when there is an underscore put two underscores but could we add a reference to where this was agreed on?